### ISSUE TITLE
Update accessibility statement Usage

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ods-ios#844]
As a user of the iOS demo app
I want to see the detailed Accessibility Conformance page
So that it is updated with the latest score

Content update:
* The item in the the app's about page to access this content should be named: "Accessibility Statement"
* The accessibility detailed page should be integrated in the App and contain the latest information

**Comment by pylapp** [comment]
Need specifications of the demo app and how this statement should be integrated.
